csrankings.csv

```diff
@@ -2688,6 +2692,7 @@ Claudio Gutierrez,Universidad de Chile,https://www.dcc.uchile.cl/cgutierr,jzeqUi
 Claudio Gutiérrez 0001,Universidad de Chile,https://www.dcc.uchile.cl/cgutierr,jzeqUi4AAAAJ
 Claudio Luis de Amorim,UFRJ,http://www.cos.ufrj.br/index.php/pt-BR/pessoas/details/18/1007-amorim,YQNnYZQAAAAJ
 Claudio Orlandi,Aarhus University,http://www.cs.au.dk/~orlandi,EVDZd5oAAAAJ
+Claudio Palazzi,University of Padova,https://www.math.unipd.it/~cpalazzi,PBpX1HAAAAAJ

```

Comment:
See above.

Reply:
See above, same comment about Math & CS at U Padova. This is a reference: https://www.math.unipd.it/en/courses/phd/

Reply:
I see two departments here.
https://www.math.unipd.it/en/courses/phd/
https://www.dei.unipd.it/home-page
Can you clarify?

Reply:
The Math (&CS) department is the major department for CS at the Univ. of Padova. However, there is also a subset of people from the DEI dept (Information Engineering) that are affiliated to the school of Engineering but are doing research, supervising PhD students and teaching in Informatics.

Reply:
Unless they can solely advise PhD students in the CS department, they are not eligible for inclusion.